dnpchallenge/iedconfig.xml

```diff
@@ -37,16 +37,20 @@
   </dnp3>
   <logic>
     <period>1s</period>
+    <process-updates>true</process-updates>
     <program>
       <![CDATA[
-      turbine.status = turbine.control == false
-      turbine = turbine.control ? 1 : 0
+      turbine = control == 1 and estop == 0
+      status = turbine
+      # hack to get estop tag to show up in Telnet module
+      estop = estop
```

Comment:
Adding in `estop` here as a way for users to manually trigger an estop on the ot-sim IED device.